prio-graph-scheduler/src/deserializable_packet.rs

```diff
@@ -0,0 +1,39 @@
+use std::collections::HashSet;
+use solana_sdk::hash::Hash;
+use solana_sdk::message::AddressLoader;
+use solana_sdk::packet::Packet;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::transaction::{SanitizedTransaction, SanitizedVersionedTransaction};
+use std::error::Error;
+
+/// DeserializablePacket can be deserialized from a Packet.
+///
+/// DeserializablePacket will be deserialized as a SanitizedTransaction
+/// to be scheduled in transaction stream and scheduler.
```

Comment:
Extract a trait from `ImmutableDeserializedPacket` itself, and let `ImmutableDeserializedPacket` implement this to avoid circular dependency